runtime/drivers/sqlite/migrations/0001.sql

```diff
@@ -7,6 +7,6 @@ CREATE TABLE instances (
 	embed_catalog BOOLEAN NOT NULL,
 	created_on TIMESTAMP NOT NULL,
 	updated_on TIMESTAMP NOT NULL,
-	env TEXT,
-	project_env TEXT
```

Comment:
The sqlite is now persistent in the staging env. So this should be a rename in a new migration file.